shared/actions/tracker.js

```diff
@@ -166,6 +167,12 @@ export function triggerIdentify (uid: string = '', userAssertion: string = ''
         incomingCallMap,
         callback: (error, response) => {
           console.log('called identify and got back', error, response)
+          if (error) {
+            // TODO(MM) figure out why we have this discrepancy
+            // The type is supposedly error.desc, but in practice we do error.raw.desc
+            dispatch({type: Constants.identifyFinished, error: true, payload: {error: error.raw && error.raw.desc || error.desc}})
```

Comment:
@chrisnojima mentioned why this is the case in #frontend and said he'll fix soon. This can handle both before and after fix (in case we're slow to update). After fix we should change this.